README.md

```diff
@@ -22,14 +22,21 @@ You can use the chart in this repository to install Monocular in your cluster.
 - [Helm and Tiller installed](https://github.com/kubernetes/helm/blob/master/docs/quickstart.md)
 - [Nginx Ingress controller](https://kubeapps.com/charts/stable/nginx-ingress)
   - Install with Helm: `helm install stable/nginx-ingress`
+
   - **Minikube/Kubeadm**: `helm install stable/nginx-ingress --set controller.hostNetwork=true`
+  
+  - **AWS**: For testing purpose, the install example below will set ELB security rule to allow connections from $MY_IP only.
```

Comment:
You are right. I don't have other providers environment setup to test, but I think it is safe to make it a generic helm command line override.